Source/EasyNetQ/Consumer/DefaultConsumerErrorStrategy.cs

```diff
@@ -58,13 +57,13 @@ public class DefaultConsumerErrorStrategy : IConsumerErrorStrategy
 
         protected void Connect()
```

Comment:
shouldn't connect be virtual then?

Reply:
why? its only function is to create `Connection`